2018-09-15-scipy-mm-nc/proposal/short.md

```diff
@@ -0,0 +1,14 @@
+The Short Description:
+
+This talk covers a tool developed for fast, interactive, browser-based
+visualization of datasets using Jupyter widgets developed for use with yt. 
```

Comment:
maybe add a phrase at the end of this sentence explaining what yt is

Reply:
I added a clause similar to what's described on the yt project page.